### ISSUE TITLE
When i hit a website offline my own wifi goes offline

### ISSUE THREAD

**Issue by sadikim1924** [the-deepnet/ddos#7]

**Comment by GoobGub** [comment]
beacause its using your computer power to send the packets witch will cause you to lag

**Comment by sadikim1924** [comment]
Is there anyway I can hit off without lagging

**Comment by GoobGub** [comment]
yes botnets or using a service that will run this code on there server but i dont recomend ur using there server power and it could be bad also this wont hit for shi just buy a botnet

**Comment by sadikim1924** [comment]
Any tips on botnets free or not that actually works?

**Comment by GoobGub** [comment]
Botnets arent free they are using a thing called putty witch is a tricky thing to setup for a beginner and they have to use a thing called a vps and server to actaully use it and ofc they have to pay for the server and vps so yeah its gonna cost no such thing as a free botnet

**Comment by sadikim1924** [comment]
Since there's no such thing as a free botnets do u know anybody that sells
them for good prices?

**Comment by GoobGub** [comment]
add me on discord ill talk to u abt it mfsshadow no hashtags

**Comment by sadikim1924** [comment]
Alr I added u

**Comment by iloveyumyumshrimpnoodles** [comment]
> they are using a thing called putty witch is a tricky thing to setup for a beginner

lmao what? since when can't a beginner use a simple GUI to connect to some server?

> ur internet provider wiill shut u down forever

probably not shut down forever, but most likely you'll be reported to the police and your contract will be cancelled

> this thing is for running for botnets

no, this thing is a simple script for running stress tests on some service, not for botnets.

also, it's copied from [this](https://github.com/MatrixTM/MHDDoS) lol

**Comment by GoobGub** [comment]
> 
i meant its not easy to get a server nor is it easy to find one that will not detect ddos attacks 

and yes in lots of places they can restrict you permanent access of internet
